tensorboard/webapp/metrics/views/main_view/card_grid_component.ts

```diff
@@ -46,14 +48,45 @@ export class CardGridComponent {
   @Output() pageIndexChanged = new EventEmitter<number>();
   @Output() groupExpansionToggled = new EventEmitter<void>();
 
-  showExpand(isBottomControl: boolean) {
+  constructor(
+    @Optional() private readonly cdkScrollable: CdkScrollable | null
+  ) {}
+
+  showExpand(isBottomControl: boolean): boolean {
     return isBottomControl ? this.isGroupExpandable : false;
   }
 
   showPaginationInput(isBottomControl: boolean) {
     return isBottomControl;
   }
 
+  PaginationCallback(pageIndex: number, target: HTMLElement) {
+    // Clear call stack to allow dom update before updating scroll to keep
+    // relative position.
+    setTimeout(
+      this.scrollToKeepTargetPosition.bind(
```

Comment:
bind() is a bit of an old fashioned way to bind a function to the correct `this`. arrow functions are our conventional way to do this now. I think this should work:

```
setTimeout(() => {
  this.scrollToKeepTargetPosition(target, target.getBoundingClientRect().top);
}, 0);
```

scrollToKeepTargetPosition(...) will have correct `this` scope when it is called.

Reply:
Well that is exciting that I have been coding long enough to have my ways become old fashioned :P. 

Changed.